packages/flutter/lib/src/material/dialog.dart

```diff
@@ -510,11 +522,25 @@ class AlertDialog extends StatelessWidget {
     const double m3ActionEndPadding = 18.0;
     const double m3ActionBottomPadding = 12.0;
 
+    Widget? iconWidget;
     Widget? titleWidget;
     Widget? contentWidget;
     Widget? actionsWidget;
+
+    if (icon != null) {
+      iconWidget = Padding(
+        padding: const EdgeInsets.fromLTRB(24.0, 24.0, 24.0, 0.0),
```

Comment:
I'd make  `contentPadding` nullable and follow the example that `titlePadding` sets. Please document similarly, see lines 259-261.

Reply:
PTAL